charts/sonarqube-dce/templates/sonarqube-application.yaml

```diff
@@ -168,12 +168,9 @@ spec:
             "-e",
             "/tmp/scripts/install_plugins.sh"]
           volumeMounts:
-            - mountPath: {{ .Values.sonarqubeFolder }}/extensions/downloads
+            - mountPath: {{ .Values.sonarqubeFolder }}/extensions/plugins
```

Comment:
I'm thinking of a scenario where a plugin was installed before in download then after in plugin.
Is there a possibility that we have the plugin twice both in download and plugins? would it conflict at start?

Reply:
I do not know, I can try and see what happens. Actually, existing installation in `extensions/downloads` should be "read" by the application, despite not being the official and documented way of installing plugins...

Reply:
I think there is no issue here, as we are only persisting that volume, after the pod restart the extensions/downloads will always be empty ?

Reply:
Side note from applicative point of view: at start, plugins are moved from download to plugins folder, so this + folder not being persisted, I think this is not possible to happen